relay-server/src/actors/processor.rs

```diff
@@ -551,13 +551,13 @@ impl EnvelopeProcessorService {
         project_cache: Addr<ProjectCache>,
         upstream_relay: Addr<UpstreamRelay>,
     ) -> anyhow::Result<Self> {
+        let geoip_lookup = match config.geoip_path() {
```

Comment:
you're right , the processing had to be set to enabled, otherwise it fails, but in [bc416d4](https://github.com/getsentry/relay/pull/2229/commits/bc416d4a283dc214012588bf1ef4495534839bd8) I make a change to make sure we can set the processing options even if the `processing.enabled` isn't set, which makes it easy to configure geoip db for the pops.